<launch>
  <node pkg="object_follower" type="object_follower_tf_broadcaster_node" name="object_follower" output="screen">

    <param name="base_frame" value="map"/>
    <param name="object_frame" value="object"/>
    <param name="range_diff_to_set_new_pose" value="0.2"/>
    <param name="yaw_diff_to_set_new_pose" value="15"/>
    <param name="tf_wait" value="1.0"/>
    <param name="goal_dist_from_obj" value="1.0"/>
    <param name="goal_frame" value="goal_to_follow"/>

  </node>
</launch>
